lib/content-services/src/lib/search/services/search-facet-filters.service.spec.ts

```diff
@@ -460,6 +471,63 @@ describe('SearchFacetFiltersService', () => {
         expect(searchFacetFiltersService.responseFacets.map(f => f.field)).toEqual(['field4', 'field1', 'Query 1', 'field2', 'field3']);
     });
 
+    it('should load category names for cm:categories facet', () => {
+        searchFacetFiltersService.responseFacets = null;
+        spyOn(categoryService, 'getCategory').and.returnValue(of({entry: {id: 'test-id-test', name: 'name'}}));
+        spyOn(categoryService, 'searchCategories').and.returnValue(of({
+                    list: {
+                        entries: [{
+                            entry: {
```

Comment:
You can define `entry` as variable and then use it for both `search` and `getCategory` I think

Reply:
Done, ty